about.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>About</title>
+</head>
+
+<body>
+  <header id="about-header">
+    <nav>
+      <div class="wrap top-menu-wrap display-none">
+        <nav class="top-menu">
```

Comment:
Great job! but it's required to follow the original design, so please make sure to decrease the height and font-size of your top Navbar to make it look more beautiful.

![image](https://user-images.githubusercontent.com/34779928/197262835-553f0b0e-c543-4c6d-9602-4d5f3445927a.png)